tools/java/src/com/twitter/bazel/checkstyle/heron_coding_style.xml

```diff
@@ -48,12 +48,14 @@ limitations under the License.
 -->
 
 <module name="Checker">
-	  <!--
+    <!--
         If you set the basedir property below, then all reported file
         names will be relative to the specified directory. See
         http://checkstyle.sourceforge.net/config.html#Checker
         <property name="basedir" value="${basedir}"/>
-          -->
+    -->
+
+    <property name="fileExtensions" value="java"/>
```

Comment:
i was attempting to use puppycrawl checkstyle to also check Python header, but failed. This is some leftover during refactoring and not needed now. will remove.